subprojects/file-watching/src/main/java/org/gradle/internal/watch/vfs/impl/WatchingVirtualFileSystem.java

```diff
@@ -256,18 +288,6 @@ private SnapshotHierarchy handleWatcherRegistryEvents(SnapshotHierarchy currentR
         return currentRoot;
     }
```

Comment:
I wonder if it would be better to inline this method. The name does not really capture what is happening here for me.

Reply:
I renamed the method.